lib/modules/kafka-connect/module/src/main/java/io/atlasmap/kafkaconnect/module/KafkaConnectValidationService.java

```diff
@@ -77,10 +77,12 @@ protected AtlasModuleDetail getModuleDetail() {
     @Override
     protected void validateModuleField(String mappingId, KafkaConnectField field, FieldDirection direction,
             List<Validation> validations) {
-        validatorMap.get("kafkaconnect.field.type.not.null").validate(field, validations, mappingId, ValidationStatus.WARN);
-
-        if (field.getPath() == null) {
-            validatorMap.get("kafkaconnect.field.path.not.null").validate(field.getPath(), validations, mappingId);
+        AtlasValidator aVal = validatorMap.get("kafkaconnect.field.type.not.null");
```

Comment:
I guess the real fix is on L62-L63, replace `json.field.type.not.null` to `kafkaconnect.field.type.not.null` and like that?